en/6-use-open-standards-solutions.md

```diff
@@ -32,12 +32,54 @@ altLangPage: 6-utiliser-normes-solutions-ouvertes
 
 <section class="dpgn-section-guideline">
 
-## 6.1 Leverage open standards and embrace leading practices
+## 6.1 Leverage open standards, use open source software and embrace leading practices
 
 <div class="dpgn-section-intro-guideline">
 
 **[TODO: Add/revise introductory text]**
 
+Give equal consideration to open source software when you choose technology.
+
+Your technology project or programme could benefit from:
+
+- solving common problems with readily available open source technology
+- more time and resource for customised solutions to solve the rare or unique problems
+- lower implementation and running costs
+
+Be aware that open source software is not completely free, so take into account the total cost of migrating, including exit and transition costs.

```

Comment:
Is "take into account the total cost of migrating" addressed in the checklist? It sounds like an action that should be taken so would be appropriate for a checklist item (if a checklist item covering this aspect doesn't already exist).

Reply:
That's a good question that we will need to submit to the Checklist team (I'm understanding here the Build-it-right checklist?)

Reply:
Build It Right is currently only a small subset of the checklist items, so wouldn't apply in this case. For changes to non-Build It Right checklist items (and adding checklist items), they would just go to the regular reviewers for the section (so the same reviewers as for the introduction).

If you are actually looking to change a Build It Right checklist item, then it should be changed through Build It Right first before changing it in the Digital Playbook.

Reply:
Sorry @pjackson28 , I was confused as I thought we were referring to the three layers of the EA.
We can definitely update the checklists here first.